can't build on Ubuntu Noble 24.04

```
# make
make -C ipt
make[1]: Entering directory '/usr/src/xt_tls/ipt'
cc -I../src -fPIC   -c -o libxt_tls.o libxt_tls.c
cc -shared -o libxt_tls.so libxt_tls.o;
/usr/bin/ld: libxt_tls.o: in function `_init':
libxt_tls.c:(.text+0x35f): multiple definition of `_init'; /usr/lib/gcc/x86_64-linux-gnu/13/../../../x86_64-linux-gnu/crti.o:(.init+0x0): first defined here
collect2: error: ld returned 1 exit status
make[1]: *** [Makefile:7: libxt_tls.so] Error 1
rm libxt_tls.o
make[1]: Leaving directory '/usr/src/xt_tls/ipt'
make: *** [Makefile:6: all] Error 2
```

* gcc-13
* kernel 6.8.0-11-generic

Thanks.

--- a/dkms.conf
+++ b/dkms.conf
@@ -4,7 +4,7 @@ CLEAN="make clean KERNELDIR=/lib/modules/${kernelver}/build KERNEL_VERSION=$kern
 BUILT_MODULE_NAME=xt_tls
 BUILT_MODULE_LOCATION=src/
 PACKAGE_NAME=xt_tls
-PACKAGE_VERSION=0.3.4
+PACKAGE_VERSION=0.3.5
 #REMAKE_INITRD=yes
 AUTOINSTALL="no"
 POST_INSTALL=dkms/post-install.sh

--- a/ipt/libxt_tls.c
+++ b/ipt/libxt_tls.c
@@ -125,7 +125,7 @@ static struct xtables_match tls_match = {
 	.x6_options	= tls_opts,
 };
 
-void _init(void)
+static __attribute__((constructor)) void xt_tls_mt_ldr(void)
 {
 	xtables_register_match(&tls_match);
 }

--- a/src/xt_tls_main.c
+++ b/src/xt_tls_main.c
@@ -450,5 +450,5 @@ module_exit(tls_mt_exit);
 MODULE_LICENSE("GPL");
 MODULE_AUTHOR("Nils Andreas Svee <nils@stokkdalen.no>");
 MODULE_DESCRIPTION("Xtables: TLS (SNI) matching");
-MODULE_VERSION("0.3.4");
+MODULE_VERSION("0.3.5");
 MODULE_ALIAS("ipt_tls");

